assets/sw.js

```diff
@@ -0,0 +1,95 @@
+"use strict";
+
+var version = 'v1.0.0::';
+console.log('Service Worker: Version', version, 'starting.');
+
+var offlineFundamentals = [
```

Comment:
We should probably cache the home page so that people with slower internet speeds can launch and load the app faster.